vivisect/parsers/elf.py

```diff
@@ -587,19 +624,27 @@ def applyRelocs(elf, vw, addbase=False, baseaddr=0):
             logger.debug('relocs: 0x%x: %s (%s)', rlva, dmglname, name)
             if arch in ('i386', 'amd64'):
                 if name:
+                    #if dmglname == 
                     if rtype == Elf.R_X86_64_IRELATIVE:
                         # before making import, let's fix up the pointer as a BASEPTR Relocation
                         ptr = r.r_addend
-                        vw.addRelocation(rlva, vivisect.RTYPE_BASEPTR, ptr)
+                        vw.addRelocation(rlva, RTYPE_BASEPTR, ptr)
                         logger.info('Reloc: R_X86_64_IRELATIVE 0x%x', rlva)
 
                     if rtype in (Elf.R_386_JMP_SLOT, Elf.R_X86_64_GLOB_DAT, Elf.R_X86_64_IRELATIVE):
                         logger.info('Reloc: making Import 0x%x (name: %s/%s) ', rlva, name, dmglname)
                         vw.makeImport(rlva, "*", dmglname)
                         vw.setComment(rlva, name)
 
-                    elif rtype in (Elf.R_386_32, Elf.R_386_COPY, Elf.R_X86_64_TPOFF64):
-                        pass
+                    elif rtype == Elf.R_386_COPY:  # Also covers X86_64_COPY
+                        # the linker is responsible for filling these in so we probably won't have these
+                        vw.addRelocation(rlva, RTYPE_BASERELOC, 0)
```

Comment:
should this just be added to the group above?  rather than a relocation to 0?
i don't know if it will make a difference either way, except that LOC_IMPORTS are known to be, well, imports.

Reply:
likewise.  consider later.